Bookish.DataAccess/DataHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+
+namespace Bookish.DataAccess
+{
+    public class DataHelper
+    {
+        private static readonly IDbConnection DbConnection = new SqlConnection(@"Server = localhost; Database = Bookish; Trusted_Connection = True;");
```

Comment:
I wouldn't recommend having this as a static variable. I would make this an instance variable and make the class have only instance methods 🙂